src/bin/rover.rs

```diff
@@ -0,0 +1,13 @@
+use rover::*;
```

Comment:
Q: Am I understanding correctly that a `bin` _folder_ have special treatment? Looks like it's treated as if it's `main.rs`?

Reply:
yup! https://doc.rust-lang.org/cargo/guide/project-layout.html

Reply:
Nice! I really should find time to read the cargo book 🦀

Reply:
fwiw it's not as long as it appears! i really wish i had to when i first started, instead i learned a lot of things the hard+slow way. highly recommend!